tasking/pulp/tasking/celery_app.py

```diff
@@ -92,6 +94,7 @@ def get_resource_manager_lock(name):
     :param name:   The hostname of the worker
     :type  name:   basestring
     """
+    from pulp.app.models.task import TaskLock, Worker
```

Comment:
I think we could do this:

```
from django.apps import apps  # yes you read that right

<other code here>

def get_resource_manager_lock(name):
    TaskLock = apps.get_app_config('pulp_app').models['tasklock']
```

This avoids doing an import, but I'm not sure it's an improvement. It requires knowing these two strings, which isn't very likable. The import seems simpler and more obvious to understand.